config/date.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/*
+ * This file is part of Bootstrap CMS.
+ *
+ * (c) Graham Campbell <graham@cachethq.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | PHP Format: NOTE this must be compatiable with the javascript date
+    |--------------------------------------------------------------------------
+    |
+    | Take a look at http://php.net/manual/en/function.date.php
+    |
+    | The default is ISO-8601
+    |
+    */
+
+    'php_format' => 'Y-m-d H:i',
```

Comment:
I feel like using an ISO standard as the default makes more sense than picking a specific region's date/time format.  It should cause less confusion for someone starting to use the project.  Is there a strong reason to use the current default?

<sub>sidenote: The relevant [XKCD](https://xkcd.com/1179/)</sub>